Library/Formula/blackbox.rb

```diff
@@ -0,0 +1,36 @@
+class Blackbox < Formula
+  homepage "http://the-cloud-book.com/"
+  url "https://github.com/StackExchange/blackbox/archive/v1.20150730.tar.gz"
+  version "1.20150730"
+  sha256 "a28237e8839d000273d0fe0ba8f9701cab66a3b942787c5c8e8d329912564ae4"
+  revision 1
+
+  depends_on "gnupg" 
+
+  def install
+    bin.install "bin/_blackbox_common.sh"
```

Comment:
Probably not. We want everything but I didn't want to potentially overwrite anything in case of an upstream change. 

> On Sep 26, 2015, at 8:58 PM, Dominyk Tiller <notifications@github.com> wrote:
> 
> In Library/Formula/blackbox.rb:
> 
> > @@ -0,0 +1,36 @@
> > +class Blackbox < Formula
> > +  homepage "http://the-cloud-book.com/"
> > +  url "https://github.com/StackExchange/blackbox/archive/v1.20150730.tar.gz"
> > +  version "1.20150730"
> > +  sha256 "a28237e8839d000273d0fe0ba8f9701cab66a3b942787c5c8e8d329912564ae4"
> > +  revision 1
> > +
> > +  depends_on "gnupg" 
> > +
> > +  def install
> > +    bin.install "bin/_blackbox_common.sh"
> Is there anything in bin/ that we don't want to install, out of curiosity?
> 
> —
> Reply to this email directly or view it on GitHub.
>